src/ontology/edits/oeo-physical.omn

```diff
@@ -3452,7 +3452,7 @@ Class: OEO_00000334
 
     Annotations: 
         OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A power generating unit is an energy transformation unit that contains a generator, among other parts.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A power generating unit is an energy transformation unit that contains a generator."@en,
```

Comment:
The `@en` at the end is not a mistake, but we should document it somewhere, whether we want to have it or not. Currently, with regard to the whole OEO, it's mixed in the definition field.